yoti/readme.txt

```diff
@@ -5,109 +5,56 @@ Tags: identity, verification, login, form, 2 factor, 2 step authentication, 2FA,
+Download the free Yoti app [IOS](https://itunes.apple.com/us/app/yoti/id983980808?ls=1&mt=8)
 
-* This plugin works with the WordPress Business Plan package.
-* You must also have downloaded the Yoti app with an ID document attached.
+== Installation ==
 
 = Step1: Installing the plugin =
 
-Import the plugin into your WordPress dashboard:
-
-1. Log on to your Wordpress Admin Dashboard: e.g. https://www.wordpress-url.org.uk/wp-admin
-2. Navigate on the left hand side to Plugins > Add New
-3. Search for Yoti and install it
-4. Activate the plugin on the same screen
+* From the “Plugins” menu search for “Yoti”,
+* click “Install Now” and then “Activate”.
 
 = Step 2: Setting up your Yoti Application =
 
-If you are a business, please create a Yoti organisation by visiting [here.](https://www.yoti.com/dashboard/login-organisations)
-
-For information on how to create an organisation please visit our [developer site.](https://www.yoti.com/developers/documentation/#introduction)
-
-Once this is completed, or for personal use please access the [dashboard](https://www.yoti.com/dashboard/login) to create a new application. Note: if you have completed your organisation steps above you will need to ensure you are within your organisation dashboard by:
-
-* Clicking your profile photo on the top right
-* Select organisation
-
-This will then lead you to create a new application.
-
-Specify the basic details of your application such as the name, description and optional logo. These details can be whatever you desire and will not affect the plugin's functionality.
-
-The Data tab - Specify any attributes you'd like users to share. You must select at least one. If you plan to allow new user registrations, we recommended choosing Given Name(s), Family Name and Email Address at a minimum.
-
-The Integration tab - Here is where you specify the callback URL. The URL can be found by going to your Yoti settings page which is located in your WordPress admin dashboard.
-
-NOTE: If you get redirected to your WordPress frontpage instead of the Admin area, simply add /wp-admin to the URL.
+Visit the Yoti Dashboard [here](https://www.yoti.com/dashboard/login) to create a new application for your organisation/business.
 
-The Key tab - Here is where your keys are generated which will be inputted into the plugin settings. You will need to download your pem file and store it somewhere safe as it will be used as part of the plugin set up.
+Specify the basic details of your application such as the name, description and optional logo. These details can be whatever you desire and will not affect the plugin’s functionality.
 
-= Step 3: Plugin Setup =
+The main page - Edit the application and set your website URL in the 'Application domain' section, e.g https:yourwebsite.com
 
-To set things up, navigate on WordPress to Settings > Yoti. You will be asked to add the following information:
+The Scenarios tab - Edit the default scenario and fill in the following sections:
```

Comment:
The Scenario tab - Scenarios are different instances where you request users for information using Yoti. Scenarios lets you have different instances of the Yoti plugin widget on your site.

-Specify a name for your scenario.
-Specify what information you want to request with this scenario.
-And finally, provide a callback URL so we know where to send your users after they have used Yoti. This URL must be a subdomain of your applications' domain.

Reply:
👍

Reply:
can we add around how we only support ONE scenario

Reply:
I didn't know the plugin can only support one scenario. You can completely ignore my comment on the Scenario tab and go with what you had before.